src/forms/FormFieldAddress/Address.test.tsx

```diff
@@ -52,6 +57,7 @@ export const AddressFormFieldExample = (props: { inputSettings?: AddressFieldDef
 			state={state}
 			fields={newFields}
 			dispatch={dispatch}
+			// buttons={renderButtons(dispatch)}

```

Comment:
I try running the tests and have these results:

![image](https://user-images.githubusercontent.com/87880265/203590714-1f63ae7c-1977-4723-932e-fb1f6950faba.png)

Could it be that the tests are failing since these buttons are not rendered?